moto/backends.py

```diff
@@ -122,3 +123,51 @@ def get_model(name, region_name):
                 models = getattr(backend.__class__, "__models__", {})
                 if name in models:
                     return list(getattr(backend, models[name])())
+
+
+def build_url_index():
+    """
+    Builds an index between the backend name and the associated url_bases patterns.
+    :rtype: Dict[str, List[str]]
+    """
+    index = dict()
+    here = os.path.dirname(__file__)
+
+    for name, (module_name, _) in BACKENDS.items():
+        file_path = os.path.join(here, module_name, "urls.py")
```

Comment:
reflecting on this now, i see that assuming you have access to the source files in this way may be problematic for scenarios like packaging moto in archives.

any thoughts?

Reply:
Yeah - maybe that's the reason TF is failing? (Don't have time to test it out myself atm..)
You could try this instead:
```
from moto.utilities.utils import load_resource
load_resource(__name__, f"{module_name}/urls.py", as_json=False)
```
https://github.com/spulec/moto/blob/d278fd6eaad3194b6d2347876d4fe24948f1cb2c/moto/utilities/utils.py#L22:L29

Rebasing with master should fix the linter - I just pushed a fix for that earlier.

Reply:
unfortunately that uses `pkgutil`, which will again cause importlib to actually load the modules, which is what we want to avoid.
it seems to me that there's no way around a build-time step that creates this index and packages it with the distribution.